### PR TITLE
ci: add github action to trigger E2E

### PR DESCRIPTION
**Pull request by Madhu-1**
based on the discussion on the slack channel. we are adding a github action
to trigger the CI jobs when a `ok-to-test` label is added on the PR.

This action is based on below github action https://github.com/peter-evans/create-or-update-comment

Sample Demo avaiable at https://github.com/Madhu-1/label-commentor-action-testing/pull/4

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Review by nixpanic — DISMISSED** [review]
Looks good to me. Please create the `ok-to-test` label and send a PR to not automatically start the jobs anymore.

**Comment by nixpanic** [comment]
> Looks good to me. Please create the `ok-to-test` label and send a PR to not automatically start the jobs anymore.

Great, the label already exists! Thanks :+1:

**Comment by Madhu-1** [comment]
@nixpanic Lets manually merge this PR to get CI job started?

**Comment by nixpanic** [comment]
> @nixpanic Lets manually merge this PR to get CI job started?

Wait until #3466 is merged?

File: .github/workflows/pull-request-commentor.yaml
Comment (nixpanic) [inline .github/workflows/pull-request-commentor.yaml:10]: can the `ci/*` branch(es) be excluded from this?

Comment (nixpanic) [reply .github/workflows/pull-request-commentor.yaml:10]: also wondering if this can be triggered after a rebase and the label is set...

Comment (Madhu-1) [reply .github/workflows/pull-request-commentor.yaml:10]: Yes we can trigger only on required branches. for that  we need to remove and re-add the label for now (we can see how that can be automated later?).

Comment (Madhu-1) [reply .github/workflows/pull-request-commentor.yaml:10]: updated to run on devel and release branch for now

**Review by nixpanic — APPROVED** [review]

**Review by humblec — APPROVED** [review]

File: .github/workflows/pull-request-commentor.yaml
Comment (Rakshith-R) [inline .github/workflows/pull-request-commentor.yaml:23]:
```suggestion
            /test all
```
Can we instead just use `all` instead of multiple comments  ?

**Review by Rakshith-R — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request-commentor.yaml
Comment (Madhu-1) [reply .github/workflows/pull-request-commentor.yaml:23]: We dont want to trigger all (some experimental) tests we want to trigger individual tests.

Comment (Rakshith-R) [reply .github/workflows/pull-request-commentor.yaml:23]: okay

**Review by Rakshith-R — APPROVED** [review]

**Comment by Madhu-1** [comment]
> /retest all

running `retest all` As we have skip CI label

**Comment by Madhu-1** [comment]
@mergifyio refresh

**Pull request merged**
